lib/virt.py

```diff
@@ -283,6 +264,11 @@ def append(self, content):
         """Append arbitrary string content to the kickstart template."""
         self.appends.append(content)
 
+    def add_pre(self, content):
+        new = ('%pre --interpreter=/bin/bash --erroronfail\n'
+               'set -xe; exec >/dev/tty 2>&1\n' + content + '\n%end')
+        self.append(new)
+
```

Comment:
The function is not used anywhere

Reply:
I know, but as the commit message says:
```
This is mainly useful for debugging and patching Anaconda.
Copied from add_post().
```
I needed to add it for debugging `/hardening/anaconda` and, if we drop the commit, it's likely I'll need to re-create it next time we need to debug that test.